### PR TITLE
feat: added poll results render to component

### PR DESCRIPTION
**Pull request by yevhendevbox**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/store/polls/actions.ts [hunk 1]
```diff
@@ -19,14 +19,27 @@ export const actions: ActionTree<PollsStateInterface, RootStateInterface> & Acti
...
+  },
 
   [PollsActionEnum.SET_SELECTED_POLL]: ({ commit }, payload) => {
     commit(PollsMutationEnum.SET_SELECTED_POLL, payload);
   },
 
   [PollsActionEnum.ADD_COOPERATION_POLL]: async ({ commit }, payload) => {
     try {
-      console.log(payload)
+      console.log(payload);
```
Comment (aleonovdementiev) [inline src/store/polls/actions.ts]: Do you need this console.log here?

File: src/views/CooperationPolls.vue [hunk 2]
```diff
@@ -54,7 +54,12 @@ export default defineComponent({
     };
   },
   mounted() {
-    this.$store.dispatch(`${StoreModuleEnum.pollsStore}/${PollsActionEnum.SET_COOPERATION_POLLS}`, this.cooperationId);
+    // this.$store.dispatch(`${StoreModuleEnum.pollsStore}/${PollsActionEnum.SET_COOPERATION_POLLS}`,
```
Comment (aleonovdementiev) [inline src/views/CooperationPolls.vue]: Do you need this commented code here?

File: src/views/CooperationPolls.vue [hunk 3]
```diff
@@ -63,7 +68,8 @@ export default defineComponent({
       ];
     },
     cooperationPolls(): Array<PollModel> {
-      return this.$store.getters[`${StoreModuleEnum.pollsStore}/${PollsGettersEnum.getPolls}`];
+      // return this.$store.getters[`${StoreModuleEnum.pollsStore}/${PollsGettersEnum.getPolls}`];
```
Comment (aleonovdementiev) [inline src/views/CooperationPolls.vue]: And here

File: src/store/polls/actions.ts (hunk 1 above)
Comment (yevhendevbox) [reply src/store/polls/actions.ts]: Good point, i missed it. Removed

File: src/views/CooperationPolls.vue (hunk 2 above)
Comment (yevhendevbox) [reply src/views/CooperationPolls.vue]: Done

File: src/views/CooperationPolls.vue (hunk 3 above)
Comment (yevhendevbox) [reply src/views/CooperationPolls.vue]: Done

**Review by anatoliy-stepanchenko — APPROVED** [review]

**Review by aleonovdementiev — APPROVED** [review]

**Pull request merged**